snap/info.go

```diff
@@ -648,12 +648,13 @@ func (slot *SlotInfo) String() string {
 type SlotInfo struct {
 	Snap *Info
 
-	Name      string
-	Interface string
-	Attrs     map[string]interface{}
-	Label     string
-	Apps      map[string]*AppInfo
-	Hooks     map[string]*HookInfo
+	Name             string
+	Interface        string
+	Attrs            map[string]interface{}
+	Label            string
+	Apps             map[string]*AppInfo
+	Hooks            map[string]*HookInfo
+	HotplugDeviceKey string
```

Comment:
Perhaps add a newline in case there will be more hot plug things here?

Reply:
Added, with a comment, thanks.